packages/cli/source/commands/convert.tsx

```diff
@@ -105,16 +105,27 @@ export const options = zod.object({
     )
     .optional()
     .default(false),
-  minIncomeBp: zod
-    .number()
+  minIncomeLimitBp: zod
+    .preprocess(val => BigInt(val as string), zod.bigint())
+    .describe(
+      option({
+        description: "Min income limit in basis points as percentage of amount",
+        alias: "income-limit",
+      }),
+    )
+    .optional()
+    .default(3n),
+  minIncomeModifier: zod
+    .preprocess(val => BigInt(val as string), zod.bigint())
     .describe(
       option({
-        description: "Min income in basis points as percentage of amount",
-        alias: "bp",
+        description:
+          "Increase the min income offered to make swaps more likely to succeed. If min income is negative this will increase the amount offered in the swap. If the min the min income is positive this will accept a lower profit.",
```

Comment:
```suggestion
          "Increase the min income offered to make swaps more likely to succeed. If min income is negative this will increase the amount offered in the swap. If the min income is positive this will accept a lower profit.",
```